src/pipeline_v3/core/postgres_base_rls.py

```
@@ -0,0 +1,379 @@
+            self._sync_pool.close()
+            self._sync_pool = None
+
+        if self._async_pool:
+            # Async pool closing needs to be done in async context
+            import asyncio
+
+            try:
+                loop = asyncio.get_event_loop()
+                if loop.is_running():
+                    # Schedule closing for later
+                    asyncio.create_task(self._close_async_pool())
```

Ruff (0.11.9)

Store a reference to the return value of `asyncio.create_task`

(RUF006)